tests/test_entry_notifier.py

```diff
@@ -0,0 +1,301 @@
+from __future__ import print_function
+
+'''----------------------------------------------------------------------------'''
+''' Copyright (c) FIRST 2017. All Rights Reserved.                             '''
+''' Open Source Software - may be modified and shared by FRC teams. The code   '''
+''' must be accompanied by the FIRST BSD license file in the root directory of '''
+''' the project.                                                               '''
+'''----------------------------------------------------------------------------'''
```

Comment:
Shouldn't this copyright header be comments before the future statement?

Reply:
Nope, __future__ needs to be before anything else (except comments), and these aren't comments. Technically, there's no reason they have to be this way, they could be converted to comments.